cli/module_registry.go

```diff
@@ -699,6 +699,13 @@ func readModels(path string, logger logging.Logger) ([]ModuleComponent, error) {
 	defer vutils.UncheckedErrorFunc(func() error { return os.RemoveAll(parentAddr) })
 	parentAddr += "/parent.sock"
 
+	// in some scenarios, we want to run module code without having a "real"
+	// parent available. This environment variable illustrates this.
+	if err := os.Setenv("VIAM_NO_MODULE_PARENT", "true"); err != nil {
+		return nil, err
+	}
+	defer os.Unsetenv("VIAM_NO_MODULE_PARENT")
```

Comment:
same issue as above, but I think this one is okay to leave unchecked:

```suggestion
        //nolint:errcheck
	defer os.Unsetenv("VIAM_NO_MODULE_PARENT")
```